build-system/tasks/runtime-test.js

```diff
@@ -60,6 +60,27 @@ function getConfig() {
   if (argv.ie) {
     return Object.assign({}, karmaDefault, {browsers: ['IE']});
   }
+  if (argv.chrome_canary && !argv.chrome_flags) {
+    return Object.assign({}, karmaDefault, {browsers: ['ChromeCanary']});
+  }
+  if (argv.chrome_flags) {
+    const flagList = argv.chrome_flags.split(',');
+    const formattedFlagList = [];
+    flagList.forEach(flag => {
+      formattedFlagList.push('--'.concat(flag));
+    });
+    const config = Object.assign({}, karmaDefault, {
+      browsers: ['Chrome_flags'],
+      customLaunchers: {
+        /* eslint "google-camelcase/google-camelcase": 0*/
+        Chrome_flags: {
+          base: argv.chrome_canary ? 'ChromeCanary' : 'Chrome',
```

Comment:
I suggest moving `argv.chrome_canary ? 'ChromeCanary' : 'Chrome'` to a variable like `chromeBase` outside this block, so it can be used as I suggest below.